### PR TITLE
Added discord.md + other changes

### PR DESCRIPTION
**Pull request by Dev1ss0**

### CODE REVIEW

**Review by kiokori-git — CHANGES_REQUESTED** [review]
pretty good, glad you updated the informations.
though, the discord tab looks.. a little off.
having a page with just a title, and also the link directly attached on the title is um..

**Comment by Dev1ss0** [comment]
True, wanted to get it there in the first place as a placeholder since the old discord link was invalid

**Comment by kiokori-git** [comment]
> True, wanted to get it there in the first place as a placeholder since the old discord link was invalid

i see

File: DISCORD.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Official [PojavLauncher Discord server](https://discord.gg/pojavlauncher-724163890803638273)
```
Comment (kiokori-git) [inline DISCORD.md]:
```suggestion
# Official PojavLauncher Discord server

the only official invite for the PojavLauncher server, beware of invites created by other users.
[invite link](https://discord.gg/pojavlauncher-724163890803638273)
```

**Review by kiokori-git — APPROVED** [review]
i approve

File: DISCORD.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Official PojavLauncher Discord server
+
+the only official invite for the PojavLauncher server, beware of invites created by other users.
```
Comment (RadiatedExodus) [inline DISCORD.md]: Fix the grammar error

**Review by RadiatedExodus — CHANGES_REQUESTED** [review]

File: DISCORD.md (hunk 2 above)
Comment (kiokori-git) [reply DISCORD.md]: i seriously don't see anything wrong in it

Comment (RadiatedExodus) [reply DISCORD.md]: The "the" at "the only" should have a capitalized T?

Comment (Dev1ss0) [reply DISCORD.md]: Aighty, done

**Review by RadiatedExodus — APPROVED** [review]

**Pull request merged**